### PR TITLE
{bio}[foss/2016b] PyNAST 1.2.2 PyCogent 1.9

### PR DESCRIPTION
**Pull request by SethosII**
This adds PyNAST and it's dependencies for the foss-2016b toolchain.

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d416a2445eacc1c1388b9fb7cc7a9752 for a full test report.

File: easybuild/easyconfigs/p/PyCogent/PyCogent-1.9-foss-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+name = 'PyCogent'
+version = '1.9'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://%(namelower)s.org'
+description = """PyCogent is a software library for genomic biology. It is a fully
+ integrated and thoroughly tested framework for: controlling third-party applications;
+ devising workflows; querying databases; conducting novel probabilistic analyses of
+ biological sequence evolution; and generating publication quality graphics."""
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyCogent/PyCogent-1.9-foss-2016b-Python-2.7.12.eb]: This is different from the existing `PyCogent` easyconfigs, but I like this a lot better.

Do you mind changing it to this in the existing `PyCogent` easyconfigs (makes sense to do that in this PR imho)?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (3 easyconfigs in this PR)
node2301.phanpy.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1ed1677eafd888918d8711bc6797e32d for a full test report.

File: easybuild/easyconfigs/p/PyNAST/PyNAST-1.2.2-foss-2016b-Python-2.7.12.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+name = 'PyNAST'
+version = '1.2.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://biocore.github.io/%(namelower)s'
+description = """PyNAST is a reimplementation of the NAST sequence aligner, which has
+ become a popular tool for adding new 16s rRNA sequences to existing 16s rRNA alignments.
+ This reimplementation is more flexible, faster, and easier to install and maintain than
+ the original NAST implementation."""
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyNAST/PyNAST-1.2.2-foss-2016b-Python-2.7.12.eb]: Same here, this is better than the values for `homepage` and `description` in the existing `PyNAST` easyconfig, do you mind changing it in the existing `PyNAST` easyconfig too?

File: easybuild/easyconfigs/p/PyCogent/PyCogent-1.9-foss-2016b-Python-2.7.12.eb (hunk 1 above)
Comment (SethosII) [reply easybuild/easyconfigs/p/PyCogent/PyCogent-1.9-foss-2016b-Python-2.7.12.eb]: @boegel I added this to the existing easyconfigs.

File: easybuild/easyconfigs/p/PyNAST/PyNAST-1.2.2-foss-2016b-Python-2.7.12.eb (hunk 2 above)
Comment (SethosII) [reply easybuild/easyconfigs/p/PyNAST/PyNAST-1.2.2-foss-2016b-Python-2.7.12.eb]: @boegel I added this to the existing easyconfigs.

**Comment by boegel** [comment]
Going in, thanks @SethosII!

**Pull request merged**